scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml

```diff
@@ -321,8 +321,19 @@ DFN-8-1EP_4x4mm_P0.8mm_EP2.3x3.24mm:
     nominal: 0.5
     maximum: 0.6
 
-  EP_size_x: 2.30
-  EP_size_y: 3.24
+  EP_size_x:
+    minimum: 2.05
+    nominal: 2.20
+    maximum: 2.30
+
+  EP_size_y:
+    minimum: 2.82
+    nominal: 3
+    maximum: 3.23
+
+  EP_size_x_override: 2.30
```

Comment:
`EP_size_x_override` should be `EP_size_x_overwrite`

Reply:
Oh, that explains it. 🤦‍♂